src/main/java/co/inventorsoft/academy/model/Team.java

```diff
@@ -0,0 +1,48 @@
+package co.inventorsoft.academy.model;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Getter
+@Setter
+@ToString(of = {"name","pilot1","pilot2"})
```

Comment:
It is always better to have `id`, in theory you can have `name`, `pilot1`, `pilot2` with exacty same values and only `id` will let you know what is what.